packages/wrangler/src/versions/upload.ts

```diff
@@ -108,9 +108,13 @@ function errIsStartupErr(err: unknown): err is ParseError & { code: 10021 } {
 	return false;
 }
 
-export default async function versionsUpload(props: Props): Promise<void> {
+export default async function versionsUpload(
+	props: Props
+): Promise<{ versionId: string | null }> {
```

Comment:
Is this used anywhere? Should there also be an output entry for version upload? in versions/index.ts

Reply:
Added

Reply:
https://github.com/cloudflare/workers-sdk/pull/6383/commits/8483a680c7e0d023c987f7d98f1df73a9c319fc8